### PR TITLE
Refactor CovidSim icdf array population into helper function

### PR DESCRIPTION
**Pull request by mstevens-uk**
A helper function setICDF() has been added to CovidSim to consolidate the population logic for icdf arrays.

### CODE REVIEW

File: src/CovidSim.cpp [hunk 1]
```diff
@@ -59,6 +59,8 @@ int GetInputParameter(FILE*, FILE*, const char*, const char*, void*, int, int, i
 int GetInputParameter2(FILE*, FILE*, const char*, const char*, void*, int, int, int);
 int GetInputParameter3(FILE*, const char*, const char*, void*, int, int, int);
 
+void setICDF(double* icdf, int startValue);
```
Comment (CloneDeath) [inline src/CovidSim.cpp:62]: Function name should be SetICDF to conform with the other PascalCase global function names.

File: src/CovidSim.cpp [hunk 2]
```diff
@@ -5665,3 +5646,16 @@ int GetInputParameter3(FILE* dat, const char* SItemName, const char* ItemType, v
 	//	fprintf(stderr,"%s\n",SItemName);
 	return FindFlag;
 }
+
+/* helper function to set icdf arrays */
+void setICDF(double* icdf, int startValue)
+{
+	int i;
```
Comment (CloneDeath) [inline src/CovidSim.cpp:5653]: Move i declaration inline to for loop.

ie: `for (int i = 0; i < CDF_RES; i++)`

File: src/CovidSim.cpp (hunk 1 above)
Comment (mstevens-uk) [reply src/CovidSim.cpp]: Agreed - change committed.

File: src/CovidSim.cpp (hunk 2 above)
Comment (mstevens-uk) [reply src/CovidSim.cpp]: Agreed - change committed.

**Pull request merged**